ui/src/components/FormModifications/FormModifications.test.tsx

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import { render, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { setUnifiedConfig } from '../../util/util';
+import {
+    firstModificationField,
+    getConfigWithModifications,
+    secondModificationField,
+    firstStandardTextField,
+    thirdModificationField,
+    secondStandardTextField,
+} from './TestConfig';
+import EntityModal, { EntityModalProps } from '../EntityModal/EntityModal';
+import { EntitiesAllowingModifications } from '../BaseFormTypes';
+import { invariant } from '../../util/invariant';
+
+const handleRequestClose = jest.fn();
+const setUpConfigWithDefaultValue = () => {
+    const newConfig = getConfigWithModifications();
+    setUnifiedConfig(newConfig);
+};
+
+const renderModalWithProps = (props: EntityModalProps) => {
+    render(<EntityModal {...props} handleRequestClose={handleRequestClose} />);
+};
+
+const props = {
+    serviceName: 'account',
+    mode: 'create',
+    stanzaName: undefined,
+    formLabel: 'formLabel',
+    page: 'configuration',
+    groupName: '',
+    open: true,
+    handleRequestClose: () => {},
+} satisfies EntityModalProps;
+
+const findMods = (
```

Comment:
probably, we don't want to return undefined here. 
If this method returns undefined, then there is something wrong with our tests, is it?

Reply:
this is a reason of scattered null-checks